utility/src/main/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsAssessorImpl.kt

```diff
@@ -70,4 +76,33 @@ class PerformanceMetricsAssessorImpl @Inject constructor(
       else -> HIGH_MEMORY_TIER
     }
   }
+
+  override fun getCurrentCpuUsageParameters(
+    currentApplicationState: ApplicationState
+  ): CpuUsageParameters {
+    return CpuUsageParameters.newBuilder().apply {
+      cpuTime = Process.getElapsedCpuTime()
+      processTime = oppiaClock.getCurrentTimeMs()
+      applicationState = currentApplicationState
+      numberOfActiveCores = getNumberOfCores()
+    }.build()
+  }
+
+  override fun getRelativeCpuUsage(
+    cpuUsageAtStartOfTimeWindow: CpuUsageParameters,
+    cpuUsageAtEndOfTimeWindow: CpuUsageParameters
+  ): Double {
+    val deltaCpuTimeMs = cpuUsageAtEndOfTimeWindow.cpuTime - cpuUsageAtStartOfTimeWindow.cpuTime
+    val deltaProcessTimeMs =
+      (cpuUsageAtEndOfTimeWindow.processTime * cpuUsageAtEndOfTimeWindow.numberOfActiveCores) -
+        (cpuUsageAtStartOfTimeWindow.processTime * cpuUsageAtStartOfTimeWindow.numberOfActiveCores)
+    return (100 * (deltaCpuTimeMs.toDouble() / deltaProcessTimeMs.toDouble()))
```

Comment:
I think we should omit the ``* 100`` (since percentages can often be easier to work with when normalized between 0 and 1, and since '100' is more for human readability which is something we can adjust on the dashboard side).

Reply:
agreed, done.